src/test/java/com/aws/greengrass/authorization/AuthorizationHandlerTest.java

```diff
@@ -117,13 +115,13 @@ private AuthorizationPolicy getStarResourceAuthZPolicy() {
                 .build();
     }
 
-    private AuthorizationPolicy getWildcardResourceAuthZPolicy() {
+    private AuthorizationPolicy getStarWildcardResourceAuthZPolicy() {
         return AuthorizationPolicy.builder()
                 .policyId("Id1")
                 .policyDescription("Test policy")
                 .principals(new HashSet<>(Arrays.asList("compA")))
                 .operations(new HashSet<>(Arrays.asList("OpA")))
-                .resources(new HashSet<>(Arrays.asList("abc*/+/*xyz*4/#")))
+                .resources(new HashSet<>(Arrays.asList("abc*/xyz*4/*")))
```

Comment:
keep some with + and #
Customers can still use those in policies, they just have a literal meaning

Reply:
You're right I should just keep them as is